public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -194,6 +198,17 @@ export class PrometheusDatasource extends DataSourceApi<PromQuery, PromOptions>
         rangeTarget.instant = false;
         instantTarget.range = true;
 
+        // Create exemplar query
+        if (target.exemplar) {
+          const exemplarTarget = cloneDeep(target);
```

Comment:
Why do we need the deep clone here? I assume the target is still being issued, and exemplars is just another query based on the original target.